runtime/compiler/env/VMJ9Server.cpp

```diff
@@ -154,6 +161,38 @@ TR_J9ServerVM::createResolvedMethodWithSignature(TR_Memory * trMemory, TR_Opaque
    return result;
    }
 
+TR_ResolvedMethod *
+TR_J9ServerVM::createResolvedMethodWithSignature(TR_Memory * trMemory, TR_OpaqueMethodBlock * aMethod, TR_OpaqueClassBlock *classForNewInstance,
+                                                 char *signature, int32_t signatureLength, TR_ResolvedMethod * owningMethod, const TR_ResolvedJ9JITServerMethodInfo &methodInfo)
+   {
+   TR_ResolvedJ9Method *result = NULL;
+   if (isAOT_DEPRECATED_DO_NOT_USE())
```

Comment:
Should we be introducing more uses of a function meant to be deprecated? Would `TR::Compilation->compileRelocatableCode()` not work here?

Reply:
It would, I just copy-pasted this code from the non-jitserver implementation of this method.
I'll change it to `compileRelocatableCode()`.